js/parts-map/HeatmapSeries.js

```diff
@@ -276,8 +276,8 @@ seriesType(
                 point.shapeArgs = {
                     x: Math.min(x1, x2) + pointPadding,
                     y: Math.min(y1, y2) + pointPadding,
-                    width: Math.abs(x2 - x1) - pointPadding * 2,
-                    height: Math.abs(y2 - y1) - pointPadding * 2
+                    width: Math.max(Math.abs(x2 - x1) - pointPadding * 2, 0),
+                    height: Math.max(Math.abs(y2 - y1) - pointPadding * 2, 0)
```

Comment:
```
if (
    Math.abs(x2 - x1) - pointPadding * 2 < 0 ||
    Math.abs(y2 - y1) - pointPadding * 2 < 0
) {
     pointPadding = 0
}
```
Hi @jon-a-nygaard ,
I thought that if we reset `pointPadding` before set width or height is better. It mean that pointPadding will be ignored if heatmap shapes is too small. what do you think?

Reply:
Thanks for your input @lanhq, but I am worried that removing the `pointPadding` altogether in these cases would be an unpredictable behavior, especially when `pointPadding` is a larger value.